packages/flutter_tools/templates/create/lib/main.dart.tmpl

```diff
@@ -84,9 +100,21 @@ class _MyHomePageState extends State<MyHomePage> {
         title: new Text(config.title),
       ),
       body: new Center(
+{{#withPluginHook}}
+        child: new Column(
```

Comment:
If we're going to add Column here we need to add comments explaining what's going on, especially if we add mainAxisSize as well.

I'm worried about having multiple versions of this template. We have four now (normal, driver, plugin, driver+plugin). We have micromanaged the creation of this template based on the usability studies; the more versions we have the more studies we're going to need to actually test them all. This doesn't scale.